src/Framework/AspNetCoreAnalyzers/src/SourceGenerators/PublicTopLevelProgramGenerator.cs

```diff
@@ -12,11 +12,12 @@ public class PublicProgramSourceGenerator : IIncrementalGenerator
 {
     private const string PublicPartialProgramClassSource = """
 // <auto-generated />
-#pragma warning disable CS1591
+/// <summary>
+/// Auto-generated public partial Program class for top-level statement apps
```

Comment:
Just because I'm aware of a StyleCop rule somewhere that warns on docs that don't end with a full stop, and I'm not sure if this would trigger it or not.

```suggestion
/// Auto-generated public partial Program class for top-level statement apps.
```